src/operators/cross.ml

```diff
@@ -41,6 +41,8 @@ object (self)
   (* This actually depends on [f], we have to trust the user here. *)
   method stype = s#stype
 
+  method self_sync = false
```

Comment:
Are you sure for this one? All the data is pre-buffered?

Reply:
crossfades need to not be self_sync to function, the source they control needs to be accelerated to compute transitions.

Reply:
Now that I'm thinking about it.. 😅